posix-configs/SITL/init/ekf2/standard_vtol

```diff
@@ -35,12 +35,16 @@ param set MPC_XY_FF 0.1
 param set MPC_Z_VEL_MAX 1.5
 param set MPC_Z_VEL_P 0.6
 param set MPC_Z_VEL_I 0.15
+param set FW_AIRSPD_MIN 14
+param set FW_AIRSPD_TRIM 20
```

Comment:
I would propose 16m/s here, 4% pitch is what SITL does and that is quite accurate with reality.